ces/commands.py

```diff
@@ -311,7 +311,7 @@ def __init__(self):
         BaseCommand.__init__(self, 'wallet')
 
     def execute(self, core, params):
-        currency = core.exchange_handle.get_currency(params['currency'])
+        currency = core.exchange_handle.get_currency(str(params['currency']).upper())
```

Comment:
Can you remove this `upper`? I get why you added and I don't think it's a bad idea but I think if this supported lower/upper case currency codes it should probable be implemented in another PR and should be supported by auto complete (e.g. `'wallet e<TAB>` should suggest `eth`).

Reply:
Makes a lot of sense, I will revert later (can't access now my laptop) and will look for the proper fix.